### PR TITLE
Add supporting components for flow

### PR DESCRIPTION
**Pull request by rufflian**
- Adds loading message component
- Adds header component
- Adds string input
- Adds json input - used at end of flow to review data
- UI updates
- Update useNetwork hook to provide a setter fn

### CODE REVIEW

File: src/components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    <HeaderNavigation>
+      <NavigationList $align={ALIGN.left}>
+        <NavigationItem>Raindrop Protocol: Creator</NavigationItem>
+      </NavigationList>
+      <NavigationList $align={ALIGN.center}>
+        <NavigationItem>
+          <Link href="/">Create Item Class</Link>
+        </NavigationItem>
+        <NavigationItem>
+          <Link href="https://long-banana.gitbook.io/raindrops/">
```
Comment (long-banana) [inline src/components/Header.tsx]: https://docs.raindropstudios.xyz/raindrops

File: src/components/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    const button = (
+      <Button size={SIZE.compact} endEnhancer={<ChevronDown size={24} />}>{selectedNetwork}</Button>
+    );
+    return setNetwork ? (
+      <StatefulPopover
+        focusLock
+        placement={PLACEMENT.bottomLeft}
+        content={({ close }) => (
+          <StatefulMenu
+            items={[
```
Comment (long-banana) [inline src/components/Header.tsx]: Add localnet too please. I'd love to be able to use this for local development too.

**Review by long-banana — APPROVED** [review]
2 tiny changes, then merge away! :)

File: src/components/Header.tsx (hunk 2 above)
Comment (rufflian) [reply src/components/Header.tsx]: This ended up being a little more involved due to the lack of localnet on the wallet adapter network options - I've switched it over to sol-kit's cluster list https://github.com/raindrop-studios/creator-ui/pull/6/commits/de5273dfe298f6671fbdd6143eb80e8d26ca03db

**Pull request merged**